File: boj/4153.cpp
#include <iostream>
using namespace std;

long long a,b,c,d;
int main()
{
    while(1){
        cin>>a>>b>>c;

        if(a == 0 && b == 0 && c == 0) break;

        a = a*a;
        b = b*b;
        c = c*c;

        if((a+b) == c) {
            cout<<"right"<<endl;
        }else if( (a+c) == b) {
            cout<<"right"<<endl;
        }else if( (b+c) == a) {
            cout<<"right"<<endl;
        }
        else cout<<"wrong"<<endl;
    }
    return 0;
}

File: boj/4153.py
import sys

while True:
    a1, b1, c1 = map(int, sys.stdin.readline().split())

    if a1 == b1 == c1 == 0:
        break

    a , b, c = 0, 0 , 0
    if a1 > b1 and a1 > c1:
        a,b,c = b1, c1, a1

    elif b1 > a1 and b1 > c1:
        a,b,c = a1, c1, b1
    else:
        a,b,c = a1,b1,c1
    

    if (a**2 + b**2) == c**2:
        print('right')
    else:
        print('wrong')

